scripts/Darkness/Darkness.gd

```diff
@@ -4,7 +4,7 @@ signal DamageArea_PlayerCollision()
 
 onready var damage_zone = $LightSource/OccluderEdge/DamageArea/DamageZone
 
-var DARKNESS_BASE_SPEED = 0.01
+export var DARKNESS_BASE_SPEED : float = 0.01
 var modifier = 10
```

Comment:
You may want to export `modifier` instead if you would prefer to deal with round numbers.

Reply:
Actually, I think I'd rather export both to have more fine-grained control over the initial darkness movement. I've updated my PR with the change.

This also made me think that maybe it would be great to have an item that pushes the darkness back a bit.

Reply:
That's definitely something to consider.

Reply:
The purpose of keeping the speed values separate, and retaining one value as a discrete factor, is to allow `modifier` to conveniently represent the darkness speed relative to the player speed on a scale of 0-128 and offset by 128.

For example, if you set the value to ~128 you will roughly match the speed of the player, which ensures that the margin for error is quite slim and difficult to recover from.  Exposing the value simply invites the notion of "this is configurable, please change it to whatever feels comfy for you".

Arguably, the factor of 0.01 is not entirely accurate at this point, but that does not imply that the value needs to be exported.